salt/modules/zypper.py

```diff
@@ -357,10 +357,12 @@ def info_installed(*names, **kwargs):
         t_nfo = dict()
         # Translate dpkg-specific keys to a common structure
         for key, value in pkg_nfo.items():
-            if six.PY2 and type(value) == str:
+            if type(value) == str:
```

Comment:
In Python 3, there is a hard line between the `str` type and the `bytes` type. I think we should clean this check up to use `if isinstance(value, six.string_types)` for consistency/compatibility between 2 and 3, but I think other than that we are ok here as all strings in Python 3 are already unicode.

In Python 3, strings are _encoded_ into bytes and bytes are _decoded_ into strings. We won't enter this if statement if python 3 gives us bytes.
```
# python3.4
>>> import salt.utils
>>> my_str = 'my str'
>>> type(my_str)
<class 'str'>
>>> my_bytes = my_str.encode('utf-8')
>>> type(my_bytes)
<class 'bytes'>
>>> new_str = my_bytes.decode('utf-8')
>>> type(new_str)
<class 'str'>
>>>
>>>
>>> import salt.ext.six as six
>>> isinstance(my_bytes, six.string_types)
False
>>> isinstance(my_str, six.string_types)
True
>>> isinstance(new_str, six.string_types)
True
```
Am I missing anything here?